projects/ngx-sub-form/src/lib/ngx-sub-form.types.ts

```diff
@@ -75,6 +75,8 @@ export type NgxSubFormOptions<ControlInterface, FormInterface = ControlInterface
   formGroupOptions?: FormGroupOptions<FormInterface>;
   emitNullOnDestroy?: boolean;
   componentHooks?: ComponentHooks;
+  // emit on this observable to mark the control as touched
+  touched$?: Observable<void>;
```

Comment:
This should help a bit with https://github.com/cloudnc/ngx-sub-form/issues/155 :)